ocs_ci/ocs/must_gather/must_gather.py

```
@@ -101,14 +101,36 @@ def search_file_path(self):
         else:
             files = GATHER_COMMANDS_VERSION[ocs_version][self.type_log]
+
+        # On EC clusters some must-gather files are named after the
+        # replicated metadata pool instead of the default block pool.
+        from ocs_ci.ocs.cluster import get_ec_metadata_pool_name
+
+        ec_pool = get_ec_metadata_pool_name()
+        if ec_pool:
+            logger.info(
+                f"EC metadata pool detected ({ec_pool}), will try EC "
+                f"pool name as fallback for must-gather file matching"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)